README.md

```diff
@@ -65,69 +65,114 @@ frameworks.
 <!-- GraphCMS Features & Recipes -->
 
 [using asset upload]: using-asset-upload
-[asset upload endpoint]:
-  https://graphcms.com/docs/content-api/assets#uploading-assets
-[using remote fields]: using-remote-fields
+[video1]: #
```

Comment:
Shouldn't we remove this until we have the video?

Reply:
They're placeholders and I reminder I need to do them! 🙃

If there's no video or demo it's not included in the column 👍